crab/ib-run-crab.sh

```diff
@@ -10,3 +10,40 @@ export CRAB_REQUEST="Jenkins_${CMSSW_VERSION}_${SCRAM_ARCH}_${BUILD_ID}"
 voms-proxy-init -voms cms
 crab submit -c $(dirname $0)/task.py
 crab status -d crab_${CRAB_REQUEST}
+
+export ID=$(id -u)
+export TASK_ID=$(grep crab_${CRAB_REQUEST} crab_${CRAB_REQUEST}/.requestcache | sed 's|^V||')
+
+# Wait a few seconds to get the resources assigned
+sleep 10
+
+echo "Keep checking job information until grid site has been assigned"
+GRIDSITE=""
+while [ "${GRIDSITE}" = "" ]
+do
+  export GRIDSITE=$(curl  -X GET  --cert "/tmp/x509up_u${ID}" --key "/tmp/x509up_u${ID}" --capath "/etc/grid-security/certificates/" "https://cmsweb.cern.ch:8443/crabserver/prod/task?subresource=search&workflow=${TASK_ID}" | grep -o "http://.*${TASK_ID}")
```

Comment:
may be change `http://.*${TASK_ID}` to `http.*/${TASK_ID}` so that it can cover https too or use `https*://.*/${TASK_ID}`?

Reply:
Sure! Nice catch